From 2392a596a7dccdf53e2a9eef58475fc426cff021 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Sat, 2 Aug 2014 06:40:42 +0200
Subject: [PATCH 201/212] Disable new devices in qemu 2.1

We do not support following devices with qemu-kvm-rhev:

- nvme
- tmp-tis
- usb-mtp

Disabling them.

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 default-configs/pci.mak       |    1 -
 default-configs/usb.mak       |    1 -
 redhat/qemu-kvm.spec.template |    1 +
 3 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/default-configs/pci.mak b/default-configs/pci.mak
index 5a6cc74..938f413 100644
--- a/default-configs/pci.mak
+++ b/default-configs/pci.mak
@@ -16,4 +16,3 @@ CONFIG_SERIAL=y
 CONFIG_SERIAL_PCI=y
 CONFIG_WDT_IB6300ESB=y
 CONFIG_PCI_TESTDEV=y
-CONFIG_NVME_PCI=y
diff --git a/default-configs/usb.mak b/default-configs/usb.mak
index 4ed8ff0..55c37f0 100644
--- a/default-configs/usb.mak
+++ b/default-configs/usb.mak
@@ -1,3 +1,2 @@
 CONFIG_USB_STORAGE_BOT=y
-CONFIG_USB_STORAGE_MTP=y
 CONFIG_USB_SMARTCARD=y
-- 
1.7.1

